code/drasil-gool/lib/Drasil/Shared/RendererClassesCommon.hs

```diff
@@ -157,6 +167,7 @@ class ValueElim r where
   valueInt :: r (Value r) -> Maybe Integer
   value :: r (Value r) -> Doc
 
+-- Again, wh are we recreating all these list functions?
```

Comment:
There is likely a single language that needs a "middle layer" to intercept these, tweak, and pass on.